modules/ossm-about-concepts-kiali.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+// about/ossm-about-concepts-assembly.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-concepts-kiali_{context}"]
+= {KialiProduct}
+
+{KialiProduct} provides visibility into your service mesh by showing you the microservices in your service mesh, and how they are connected.
```

Comment:
What does the `{KialiProduct}` template result in? Because if it is "Kiali Operator provided by Red Hat" then this is a little odd to me. For example, look [here at the TP1 docs](https://docs.openshift.com/service-mesh/3.0.0tp1/kiali/ossm-kiali-assembly.html#ossm-kiali-about_ossm-kiali-assembly) where it says,
```
Kiali Operator provided by Red Hat is the management console for Red Hat OpenShift Service Mesh
``` 
That isn't correct. The Operator isn't the management console - the Server is.

The same thing here - if `{KialiProduct}` results in `Kiali Operator provided by Red Hat` then it really isn't true that `Kiali Operator provided by Red Hat provides visibility into your service mesh by showing you the microservices in your service mesh, and how they are connected.` --- that's the Server, not the Operator.

Reply:
Then I need to know what other attributes for Kiali are needed. When I have tried to just use "Kiali," doc peer reviewers have come back with "{KialiProduct} is the attribute so use that here."

Reply:
I don't know how you want to do it using this doc-ese template stuff, but it should result in something like:
```
Kiali provides visibility into your service mesh by showing you the microservices in your service mesh, and how they are connected
```
In short, it isn't the OPERATOR that does all that - its the Kiali Server (or just plain "Kiali") - again, not sure how it should be worded, I just know how it should NOT be worded :)